aws-java-sdk-dynamodb/pom.xml

```diff
@@ -15,6 +15,13 @@
 
   <!-- The dependencies section in pom.xml is auto generated. No manual changes are allowed -->
   <dependencies>
+
+    <dependency>
+        <groupId>javax.annotation</groupId>
```

Comment:
The platform build bans `javax.*` and will fail.

The `javax.*` Maven artifacts are deprecated. They've been replaced wholesale with `jakarta.*` artifacts.

Use this instead:
```
<dependency>
    <groupId>jakarta.annotation</groupId>
    <artifactId>jakarta.annotation-api</artifactId>
    <version>1.3.5</version>
</dependency>
```